routes/web.php

```diff
@@ -1,7 +1,10 @@
 <?php
 
 use App\Http\Controllers\PublicController;
+use Illuminate\Http\RedirectResponse;
```

Comment:
also it don't prefer adding use of classes without the need for it, it gives me an idea that you are using this class so i scroll down to see where you use it 

if you find this wrong just forget about it, you don't have to follow my conventions